src/pkgcore/ebuild/ebd.py

```diff
@@ -211,7 +211,8 @@ def _set_op_vars(self, tmp_offset):
         for x, y in (("T", "temp"),
                      ("WORKDIR", "work"),
                      ("D", "image"),
-                     ("HOME", "homedir")):
+                     ("HOME", "homedir"),
+                     ("PKGCORE_EMPTYDIR", "empty")):
```

Comment:
Wouldn't this better be named `__PKGCORE_EMPTYDIR`, since it's an internal variable?

Reply:
I don't think the base tmpdir is exported in any fashion so using a separate variable is required for now in that sense to support it cleanly rather than recreating it on the ebd side using a different var and appending ``../empty``.

Wrt the ``__`` prefix, pkgcore doesn't use the same approach as portage and automatically filters all ``PKGCORE_`` prefixed-vars from the exported env, so it should be fine as it is now.